### PR TITLE
Add emacs syntax file matching our guidelines

### PR DESCRIPTION
**Pull request by ktf**

### CODE REVIEW

**Comment by ktf** [comment]
This was requested by @pzhristov at some point. While I am not 100% sure it follows our conventions completely, it's should be a good start. Since I use `vim`, I am not sure I can help customising this, though.

**Comment by ktf** [comment]
@sawenzel the errors comes from your stuff, actually. Can you take a look?

File: Utilities/Tools/README.md [hunk 1]
```diff
@@ -42,3 +42,10 @@ false positives. In case your false positives come from ROOT and
 boost, you can use the `Utilities/Tools/boost-root.supp` file (notice
 that different platforms and version of the software might require
 adjustments in order to eliminate all the false positives).
+
+# Configuring emacs to format according ALICE Coding guidelines
+
+While not 100% ok, the file `Utilities/Tools/google-c-style.el` closely matches
+our coding guidelines and can be used by emacs users as a template for their
```
Comment (sawenzel) [inline Utilities/Tools/README.md]: I wouldn't say coding guidelines but rather "formatting" guidelines or similar.

**Comment by Barthelemy** [comment]
I don't see anything obviously wrong. If there are fans of emacs, they can then modify this file to make it more compliant with O2 if and when needed.

File: Utilities/Tools/README.md (hunk 1 above)
Comment (ktf) [reply Utilities/Tools/README.md]: Fixed.

**Pull request merged**